src/async_fuse/memfs/fs_util.rs

```diff
@@ -328,12 +331,12 @@ pub async fn load_dir_data(dirfd: RawFd) -> anyhow::Result<BTreeMap<String, DirE
         }
         Ok(dir_entry_map)
     })
-    .await
+    .await?
```

Comment:
why ? is needed?

Reply:
to unwrap result from potential JoinError given by spawn_blocking